sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerTestBase.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Core.TestFramework;
+
+namespace Azure.Communication.CallingServer.Tests
+{
+    public class CallingServerTestBase
+    {
+        protected const string dummyAccessKey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9+eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ+SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV+adQssw5c=";
```

Comment:
Where does this access key come from?

Reply:
It's not been used anywhere, I removed it.